cmd/tailscale/main.go

```diff
@@ -240,6 +242,7 @@ func (a *App) runBackend() error {
 		rcfg *router.Config
 		dcfg *dns.OSConfig
 	}
+	interfaces.RegisterInterfaceGetter(a.getInterfaces)
```

Comment:
Can you try doing this line earlier in func main, before the goroutine to start `runBackend`?

I think it's fine here, but I can't convince myself there's not a race from something else.